### PR TITLE
Update footer and spacing

### PR DESCRIPTION
**Pull request by haowie-dev**
This ass

### CODE REVIEW

**Comment by epicchewy** [comment]
can we drop some screenshots of the updates here from time to time?

**Comment by haowie-dev** [comment]
@epicchewy git clone 
start node bruh

File: public/css/contact.css [hunk 1]
```diff
@@ -22,11 +22,13 @@
 
 .footer {
   width: 100vw;
-  height: 100px;
+  height: 120px;
+  margin: 40px 0px 0px 0px;
```
Comment (michaelnyu) [inline public/css/contact.css]: i assume this needs to be scaled for mobile

File: public/css/contact.css [hunk 2]
```diff
@@ -36,7 +38,7 @@
     .footer .links a {
       flex: 1; }
   .footer .logo {
-    margin: 20px 0px 0px 0px;
+    margin: 30px 0px 0px 0px;
```
Comment (michaelnyu) [inline public/css/contact.css]: need to be scaled probably maybe

File: public/sass/main.scss [hunk 3]
```diff
@@ -39,38 +46,48 @@ h3{
...
 	align-items: center;
 	justify-content: center;
+
+	-o-transition:.3s;
+    -ms-transition:.3s;
+    -moz-transition:.3s;
+    -webkit-transition:.3s;
+}
+
+.button:hover {
```
Comment (michaelnyu) [inline public/sass/main.scss]: i thought we changing the buttons to divs

File: views/team.html [hunk 4]
```diff
@@ -58,21 +56,15 @@
...
 
-  </section>
+  <div class="partners">
+    <h3>PARTNERS</h3>
+  </div>
   
     <div class="footer">
       <div class="logo">
         <span id="navy"> BRUIN</span><span> ENTREPRENEURS </span>
+        <div class="love">Made with <i class="fa fa-heart-o" aria-hidden="true"></i> in Los Angeles </div>
```
Comment (michaelnyu) [inline views/team.html:67]: what the f is    fa fa-heart-o

File: public/sass/footer.scss [hunk 5]
```diff
@@ -1,10 +1,12 @@
 .footer {
   width: 100vw;
-  height: 100px;
+  height: 120px;
+  margin: 40px 0px 0px 0px;
```
Comment (michaelnyu) [inline public/sass/footer.scss:4]: margin and height on footer prob need to be scaled for mobile

File: public/sass/contact.scss [hunk 6]
```diff
@@ -6,10 +6,19 @@
 		.title{
 			.inner-text-container{
 				width: 100%;
+
+				.apps {
+					margin: 40px 0 60px 0;
```
Comment (michaelnyu) [inline public/sass/contact.scss:12]: this prob need to be scaled for mobile too

**Review by michaelnyu — APPROVED** [review]
Good changes I'm getting learned abt spacing.  looks good

**Comment by epicchewy** [comment]
aint tryna run the app locally all the time just drop in some changes with some comments so everyone knows whats going on

File: views/team.html (hunk 4 above)
Comment (haowie-dev) [reply views/team.html:67]: Thats font awesome, which is where the heart icon comes from when you open it up

**Pull request merged**